src/rpc/blockchain.cpp

```diff
@@ -2346,7 +2347,7 @@ static UniValue getlicensedminers(const JSONRPCRequest& request)
     for (const auto& license : minerLicenses.GetLicenses()) {
     	UniValue licenseObj(UniValue::VOBJ);
     	licenseObj.pushKV("height", license.height);
-    	licenseObj.pushKV("hashrate", license.hashRate);
+    	licenseObj.pushKV("hashrate", license.hashRate * 10);
```

Comment:
The minimum assigned unit should be 1 PH, not 10

Reply:
Done